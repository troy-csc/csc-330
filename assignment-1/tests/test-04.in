TRACY
DALLAS
DAKOTA
KIMBERLY
YUTAKA
