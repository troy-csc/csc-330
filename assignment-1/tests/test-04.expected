Read 4340 babies. Starting year 1920. Each baby has 100 entries.
TRACY
 Total: 3826
 Years: 52
 2019: 0
 First: 1953 11
 Last: 2006 8
 Min: 2001 5
 Max: 1970 252
 Avg: 38.26
DALLAS
 Total: 1157
 Years: 63
 2019: 11
 First: 1943 5
 Last: 2019 11
 Min: 1943 5
 Max: 1994 54
 Avg: 11.57
DAKOTA
 Total: 1056
 Years: 32
 2019: 7
 First: 1988 6
 Last: 2019 7
 Min: 2012 5
 Max: 1995 88
 Avg: 10.56
KIMBERLY
 Total: 4113
 Years: 61
 2019: 0
 First: 1955 6
 Last: 2018 5
 Min: 1957 5
 Max: 1980 166
 Avg: 41.13
YUTAKA
 Total: 12
 Years: 2
 2019: 0
 First: 1930 7
 Last: 1931 5
 Min: 1931 5
 Max: 1930 7
 Avg: 0.12
